plugin/secrets_service_account_key.go

```diff
@@ -372,15 +382,15 @@ func getCacheCollection(ctx context.Context, s logical.Storage, keyName string)
 	}
 
 	if cachedKeyCollection != nil {
```

Comment:
Shouldn't this be `cachedKeyCollection == nil`?

Reply:
Right, my bad.